internal/core/workers/runtime_watcher_worker/runtime_watcher_worker_test.go

```diff
@@ -34,18 +34,17 @@ import (
 
 	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/require"
-	runtimemock "github.com/topfreegames/maestro/internal/adapters/runtime/mock"
 	"github.com/topfreegames/maestro/internal/core/entities"
 	"github.com/topfreegames/maestro/internal/core/entities/game_room"
 	porterrors "github.com/topfreegames/maestro/internal/core/ports/errors"
 	mockports "github.com/topfreegames/maestro/internal/core/ports/mock"
 	"github.com/topfreegames/maestro/internal/core/workers"
 )
 
-func workerOptions(t *testing.T) (*gomock.Controller, *runtimemock.MockRuntime, *mockports.MockRoomManager, *workers.WorkerOptions) {
+func workerOptions(t *testing.T) (*gomock.Controller, *mockports.MockRuntime, *mockports.MockRoomManager, *workers.WorkerOptions) {
 	mockCtrl := gomock.NewController(t)
 
-	runtime := runtimemock.NewMockRuntime(mockCtrl)
+	runtime := mockports.NewMockRuntime(mockCtrl)
```

Comment:
```suggestion
	runtime := mockports.NewRuntime(mockCtrl)
```

Reply:
@arthur29 I prefer to keep the scope of this PR to just update package names and directories, renaming functions would break this PR scope. Besides that, this function "NewMockRuntime" is auto generated by ou lib `mockgen`, and it is following  the standard mock constructor naming that we are using in all of our mocks.